engine/include/Engine/Core/Game/ContextStartup.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2021 Amara Sami, Dallard Thomas, Nardone William, Six Jonathan
+ * This file is subject to the LGNU license terms in the LICENSE file
+ *	found in the top-level directory of this distribution.
+ */
+
+#pragma once
+
+class ContextStartup
+{
+  public:
+    // Loads and initializes libraries
+    ContextStartup()
+    {
+    }
+
+    // Contains gameloop
+    void run()
+    {
+        for (;;)
+        {
+            update();
+        }
+    }
```

Comment:
I don't know if it's here but maybe integrate time system here without infinite loop

Reply:
In cpp or inl